client/game/Timer.jsx

```diff
@@ -1,9 +1,23 @@
 import React from "react";
 
 import { StageTimeWrapper } from "meteor/empirica:core";
+
 import Timer from "./Timer.jsx";
 
 class timer extends React.Component {
+  startTimer = () => {
+    const { remainingSeconds } = this.props;
+    const timer = remainingSeconds;
+    let minutes;
+    let seconds;
+    minutes = parseInt(timer / 60, 10);
+    seconds = parseInt(timer % 60, 10);
```

Comment:
You can out initialisation on same line as declaration here.

```suggestion
    let minutes = parseInt(timer / 60, 10);
    let seconds = parseInt(timer % 60, 10);
```